src/aims_ui/upload_utils.py

```diff
@@ -0,0 +1,38 @@
+import os
+import csv
+
+ALLOWED_EXTENSIONS = {'csv'}
+
+
+def allowed_file(filename):
+  return '.' in filename and \
+   filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+
+
+def check_for_blank_fields(file):
+  contents = file.readlines()
+  file.seek(0)
+
+  for row in contents:
+    row = row.strip().decode('utf-8').split(',', maxsplit=1)
+    row = [x.replace(',', '') for x in row]
+    if '' in row:
+      return row
```

Comment:
This could potentially made a bit more efficient and neat using the `any()` function.

```python

rows = [[cell.replace(',', '') for cell in row.strip().decode('utf-8').split(',', maxsplit=1)] for row in contents]
return any('' in row for row in rows)
```
Or something similar - I haven't tested this extensively. Split the rows line up however is needed, but if possible, don't re-use variable names.